vocabularies-gsq/site-relationships.ttl

```diff
@@ -113,9 +259,7 @@ strel:temporal a skos:Collection ;
 
 strel:wellbore a skos:Collection ;
     skos:definition "Defines the parent-child relationship of wellbores that constitute a borehole."@en ;
-    skos:member strel:parent,
-        strel:boreholeIntersect,
-        strel:injectionExtractionPair ;
+    skos:member strel:parent ;
     skos:prefLabel "Wellbore"@en .
```

Comment:
```suggestion
    skos:member strel:parent,
        strel:boreholeIntersect,
        strel:injectionExtractionPair ;
    skos:prefLabel "Wellbore"@en .
```

Reply:
It looks OK to me. Was this all that was wanted? Do you want me to commit or confirm?

Reply:
If you commit that i can do the approval and we can push this one through.
If @BlakePaul is not available can we get a review on the relationships by @LukeHauck or @GSQ-AI ?